src/Serialize.h

```diff
@@ -25,6 +31,15 @@ namespace drafter {
         UnknownASTType = -1
     };
 
+#ifdef _WITH_REFRACT_
+    /**
+     * Hold all **Named Types** converted to Refract Element
+     * Later use this registry for expanding element and rendering before serialization
+     */
+
+    static refract::Registry NamedTypesRegistry;
```

Comment:
Yes, we need it in more than one file since we need to do boutique rendering for both APIB AST and APIB Refract. Not having static is giving me "duplicate symbols".